app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  background: image-url('confectionary.png') repeat;
+  background-attachment: fixed;
+}
+
+.logo {
+  margin-right: 10px;
+}
+
+.navbar {
+  margin-right: 23px;
+  margin-top: 35px;
+}
+
+#item-title {
+  font-weight: bold;
+  margin-top: 20px;
+  margin-left: 30px;
+  text-shadow: 0px 2px 3px #666;
+}
+
+#item-description {
+  min-height: 25px;
+  font-size: 20px;
+  margin-left: 30px;
+}
+
+#item-price {
+  min-height: 24px;
+  font-size: 20px;
+  margin-left: 30px;
+}
+
+#item-button {
+  margin-left: 30px;
+}
+
+.category {
+  padding-left: 10px;
```

Comment:
Properties should be ordered list-style-type, margin-left, margin-right, padding-left, padding-right